pandas/io/html.py

```diff
@@ -198,11 +206,19 @@ class _HtmlFrameParser:
     displayed_only : bool
         Whether or not items with "display:none" should be ignored
 
+    .. versionadded:: 1.5.0
+
+    extract_links : "all"/"header"/"body"/"footer" or None, default None
+        Table elements in the specified section(s) with <a> tags will have their
+        href extracted. Note that specifying "header" will result in a
+        :class:`~pandas.MultiIndex`.
+
```

Comment:
```suggestion
    extract_links : {None, "all", "header", "body", "footer"}
        Table elements in the specified section(s) with <a> tags will have their
        href extracted. Note that specifying "header" will result in a
        :class:`~pandas.MultiIndex`.
        
        .. versionadded:: 1.5.0
```
when providing a set of possible values the default is assumed to be the first, at least numpy documents this in their style guide.